README.md

```diff
@@ -52,33 +114,47 @@ task :ci => ['engine_cart:generate'] do
 end
 ```
 
-And in your e.g. spec\_helper.rb (or rails\_helper.rb), initialize EngineCart:
+And in your engine's test framework configuration (e.g. spec\_helper.rb or rails\_helper.rb), initialize EngineCart:
 
 ```ruby
   require 'engine_cart'
   EngineCart.load_application!
 ```
 
-When you want to clean out and regenerate the test application, you can call:
+Your test files (e.g. spec files for your engine) can now be written as tests for a Rails application.
+
+## Gotcha
+
+If you have generated a test application, there is a Gemfile and Gemfile.lock associated with the testing app at spec/internal (or wherever you designated as the test app location).  If you then update *your* engine's Gemfile or gemspec, Bundler can get confused if it has conflicting information between your engine and the testing app.
+
+To fix this:
+
+1. clean out your testing app `$ rm spec/internal`  # rake may not be avail.
```

Comment:
I took a stab at a way to resolve conflicts between the gem and the app:

https://github.com/cbeer/engine_cart/pull/31